inc/features/blocks/namespace.php

```diff
@@ -286,38 +346,36 @@ function get_views( string $block_id, ?int $post_id = null ) {
 	$key = sprintf( 'views:%s:%s', $block_id, $post_id );
 	$cache = wp_cache_get( $key, 'altis-xbs' );
 	if ( $cache ) {
-		return $cache;
+		// return $cache;
```

Comment:
assuming we don't want to leave commented out code? 🧐

Reply:
ah, yeah I'd left it commented from the demo. Will fix